.gitignore

```diff
@@ -44,3 +44,4 @@ docker*.yml
 .cache
 .vscode
 src/olympia/discovery/strings.jinja2
+static/js/i18n/*.js
```

Comment:
nit, but move it next to the two other `static/` gitignore lines above? (not that we have any order or logic to the file, I know)

Reply:
edb7d63e7b0679c5128f516a04c5ed8824ac7d95